assets/webpack.config.dev.js

```diff
@@ -22,17 +22,17 @@ module.exports = env =>
         "Access-Control-Allow-Origin": "*"
       },
       devMiddleware: {
-        writeToDisk: path => /\.css$/.test(path)
+        writeToDisk: true
```

Comment:
This changes makes Webpack write everything to `/priv/static` in dev mode, as opposed to only the CSS